validators_instructions.md

```diff
@@ -0,0 +1,139 @@
+
+The Sommelier chain is a Cosmos SDK application chain. It is the core of Sommelier that makes decisions about whether to act on DP recommendations and executes Cellar functions. Transactions and function calls are only settled on-chain when the Validators reach consensus. In order to fulfill a strategy, Validators need a way to receive recommendation data from the Data Providers, reach consensus on whether to act on that data, and then call functions on the Cellars with the provided data as parameters. This is where Steward comes in.
+
+### **Steward**
+
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Data Providers send recommendations as soon as market data becomes available. This payload contains identifying information about the Cellar which Steward is meant to act on, and the recommended arguments for its rebalance function parameters. When Steward receives a new recommendation, it submits a vote to rebalance the appropriate Cellar based on that data. If Validators reach a quorum, the process of rebalancing begins.
+
+Since Sommelier is a Cosmos chain, in order to provide strategies on Ethereum it needs a way to bridge assets and logic calls. Steward is responsible for running the Orchestrator, which handles relaying Cosmos transactions to Ethereum, and co-processing Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage Cellars on Ethereum.
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. First, ensure the sommelier chain is running. Next, create a `toml` file in the root of the application which will hold your configuration. To get a template for your configuration file, run the command below:
+
+```rust
+steward -- print-config
+```
+
+Replace the default keys in the template displayed in your terminal with your configuration. Now you have the configurations set up, let’s go through stewards commands.
+
+## Start Cosmos Mode
+
+To start the Allocation signer mode, ensure that the server section in your config file is set properly as shown below. The `address`, `client_ca_cert_path` and `port` are optional fields.
+
+```rust
+keystore = "/tmp/keystore"
+
+[server]
+address = "127.0.0.1"           // This is an optional feild.
+client_ca_cert_path = ""       // optional, defaults to VolumeFi client cert
+port = 9999                   // optional, default is 5734
+server_cert_path = ""
+server_key_path = ""
+
+[gravity]
+contract = "0x0000000000000000000000000000000000000000"
+fees_denom = "stake"
+
+[ethereum]
+key_derivation_path = "m/44'/60'/0'/0/0"
+rpc = "http://localhost:8545"
+gas_price_multiplier = 1.0
+blocks_to_search = 5000
+
+[cosmos]
+key_derivation_path = "m/44'/118'/0'/0/0"
+grpc = "http://localhost:9090"
+prefix = "cosmos"
+msg_batch_size = 5
+gas_adjustment = 1.0
+
+[cosmos.gas_price]
+amount = 0.001
+denom = "stake"
+
+[metrics]
+listen_addr = "127.0.0.1:3000"
+
+[[cellars]]
```

Comment:
It might be worth pointing out that these cellar configurations would only be required in the single signer testing mode (and we should verify that they are all still necessary or if we can remove any of them).

Reply:
Can you speak with @zmanian concerning this? He was saying although some of the fields are not necessary, the config should be the same for allocation mode and test mode.